read_input_parser_cvc.l

```diff
@@ -693,25 +755,25 @@ static inline double fltlist_next_token(int * const list_end){
       }
     printf("Token=%s\n",token);
     if( strcmp(token, "Naive") == 0 ){
-      cvc::g_spinprojection_loop_type[index]=0;
+      cvc::g_LoopExtract_FilterLoopTypes[index]=0;
```

Comment:
Could you use an enum for the numeric values please? 

-> ```include/enums.hpp```

```
typedef enum LoopExtractLoopType_t {
  LOOP_EXTRACT_LOOP_TYPE_NAIVE = 0,
  LOOP_EXTRACT_LOOP_TYPE_SCALAR,
  LOOP_EXTRACT_LOOP_TYPE_DOP,
  LOOP_EXTRACT_LOOP_TYPE_LOOPS,
  LOOP_EXTRACT_LOOP_TYPE_LPSDW,
  LOOP_EXTRACT_LOOP_TYPE_LOOPSCV,
  LOOP_EXTRACT_LOOP_TYPE_LPSDWCV,
  LOOP_EXTRACT_LOOP_TYPE_NLOOPTYPES
} LoopExtractLoopType_t;
```

That way it's always more or less clear what's going on. If you're very motivated you can even add comments to the above :)